fms/modules/attention.py

```diff
@@ -265,29 +265,35 @@ def __init__(
         assert (
             nheads % world_size == 0
         ), "The number of heads must be divisible by world size"
+        assert (
+            kvheads % world_size == 0 or world_size % kvheads == 0
```

Comment:
this assert is only True if kvheads == world_size as written. The correct assert is:

```
(kvheads >= world_size and kvheads % world_size == 0) or (kvheads < world_size and world_size % kvheads == 0)```